test/spec.js

```diff
@@ -0,0 +1,33 @@
+const { Application } = require('spectron');
+const electronPath = require('electron');
+const path = require('path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const mocha = require('mocha');
+
```

Comment:
Definitely need to add more tests. From what I can tell, this only tests for if the app actually opens and connects. We'll need unit tests, tests for checking if the urls are valid, tests for displays actually showing up, and other kinds of tests that don't come to mind at the moment.